README.md

```diff
@@ -4,6 +4,9 @@
 
 This is a basic example that helps to scan uploaded resources for malwares with clamd library.
 
+Note: This interfaces via the iUploader system and needs to be first but does not return an object allowing 
```

Comment:
Perhaps something like:

`Note: This plugin provides a custom IUploader implementation to intercept and scan uploaded files.
However, the implementation does not return an object, allowing the actual IUploader implementation to take effect. If using an actual custom uploader such as ckanext-s3filestore, 'clamav' needs to come earlier in the configured plugin order.`